src/index.js

```diff
@@ -28,9 +28,13 @@ const getUrlString = url => {
   }
 };
 
-export default async ({ cache, markdownAST }) => {
+export default async ({ cache, markdownAST }, options = {}) => {
```

Comment:
Yes, I thought about that, but I didn't do it because all existing tests call the plugin with no options argument. I did some real tests with Gatsby, and it sends an empty object by default, though.

There are eight calls like this in the tests:

```javascript
    const processedAST = await plugin({ cache, markdownAST });
```
We could change them all to send an empty object and have a simple destructuring:
```javascript
export default async ({ cache, markdownAST }, { customTransformers = [] }) => {
//...
```

 Or we could destructure including a default value for the argument, without needing to change the existing test calls to `plugin`:

```javascript
export default async (
  { cache, markdownAST },
  { customTransformers = [] } = {}
) => {
//...
```

What do you prefer?

Reply:
I think setting both a default value for `options` object and `customTransformers` is the way to go 🙂